infra/modules/azure_app_service_exposed/locals.tf

```diff
@@ -9,6 +9,10 @@ locals {
     zone_balancing_enabled = local.tier != "s" && local.tier != "xs"
     is_slot_enabled        = local.tier == "s" || local.tier == "xs" ? 0 : 1
     always_on              = local.tier == "xs" ? false : true
+    command_line = {
+      node = null
```

Comment:
question: is there a way to owerwrite this for the module user?

Reply:
Not in this PR, but I think @christian-calabrese could bring the changes of the #188 (still to resolve)

Reply:
We can surely do it, I'll integrate it in this PR